cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/TotalBidMatrixCalculationInput.view.yaml

```diff
@@ -40,3 +40,44 @@ properties:
     name: partialBidMatrices
     description: The partial bid matrices that are used to calculate the total bid matrix.
     connectionType: multi_edge_connection
+  marketConfiguration:
+    container:
+      space: '{{powerops_models}}'
+      externalId: FunctionData
+      type: container
+    containerPropertyIdentifier: direct2
+    name: marketConfiguration
+    description: The market configuration to be used to generate the total bid matrix
+    source:
+      space: '{{powerops_models}}'
+      externalId: MarketConfiguration
+      version: '{{version}}'
+      type: view
+  priceArea:
```

Comment:
I also added price-area, however, it was solely used in the external ID of the created total bid matrix. So do we really need price-area as an input?

Reply:
No, then I'd remove it